src/components/SQForm/useFormButton.tsx

```diff
@@ -1,25 +1,23 @@
 import React from 'react';
 import {DebouncedState} from 'use-debounce/lib/useDebouncedCallback';
 import {useDebouncedCallback} from 'use-debounce';
-import {useFormikContext} from 'formik';
+import {FormikContextType, useFormikContext} from 'formik';
 import {hasUpdated} from '../../utils';
 
-type UseFormButtonReturnType = {
+type UseFormButtonReturnType<Values> = {
   isButtonDisabled: boolean;
   hasFormBeenUpdated: boolean;
-  values: unknown;
```

Comment:
No need to specify the `values` type because it's included in `FormikContextType`